wallet/src/wallet-api/main.rs

```diff
@@ -1779,38 +1779,17 @@ mod tests {
                 "newwallet/{}/{}/name/{}",
                 server.get::<String>("getmnemonic").await.unwrap(),
                 base64("my-password".as_bytes()),
-                base64("named_keystore".as_bytes()),
+                base64(keystore_name.as_bytes()),
             ))
             .await
             .unwrap();
         assert_eq!(
-            vec![String::from("named_keystore")],
+            vec![String::from(keystore_name)],
             server.get::<Vec<String>>("listkeystores").await.unwrap()
         );
 
-        // Create a key store by path, in the directory containing named keystores.
-        server
-            .post::<()>(&format!(
-                "newwallet/{}/{}/path/{}",
-                server.get::<String>("getmnemonic").await.unwrap(),
-                base64("my-password".as_bytes()),
-                server.path()
-            ))
-            .await
-            .unwrap();
-        let from_server_vec = server.get::<Vec<String>>("listkeystores").await.unwrap();
-        let expected: HashSet<String> =
-            vec![String::from("named_keystore"), String::from("test_wallet")]
-                .into_iter()
-                .collect();
-
-        assert_eq!(
-            expected,
-            HashSet::from_iter(from_server_vec.iter().cloned())
-        );
-
```

Comment:
Does this no longer work? Or does it just not really make sense as a test anymore?

Reply:
It doesn't work (because when listing we skip files that aren't valid tagged base 64) and it doesn't make sense anymore (because this behavior isn't part of the interface/spec of `newwallet`/`listkeystores`, and never was, really)